godot-core/src/builtin/projection.rs

```diff
@@ -337,8 +375,10 @@ impl Projection {
     /// has the given horizontal field of view (in degrees) and aspect ratio.
     ///
     /// _Godot equivalent: Projection.get_fovy()_
-    pub fn fovy_of(fov_x: real, aspect: real) -> real {
-        real::from_f64(InnerProjection::get_fovy(fov_x.as_f64(), aspect.as_f64()))
+    pub fn fovy(fov_x: real, aspect: real) -> real {
+        real::from_f64(rad_to_deg(
+            atan(aspect.as_f64() * tan(deg_to_rad(fov_x.as_f64()) * 0.5)) * 2.0,
+        ))
```

Comment:
Could you split this into multiple sub-expressions, with expressive variable names? That would help understandability of this part.